user_guide_src/source/database/utilities/001.php

```diff
@@ -1,13 +1,15 @@
 <?php
 
-class MyModel extends \CodeIgniter\Model
-{
-    protected $table      = 'foo';
-    protected $primaryKey = 'id';
-}
+$db     = db_connect();
+$dbutil = \CodeIgniter\Database\Config::utils();
```

Comment:
Can we use `Config\Database::utils()` instead?

Reply:
Oh, thanks! Fixed